test/browser_tests/page_objects/page_about-director.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+function _getQAElement( selector, getAllFlag ) {
```

Comment:
Can we pull this out to a utils folder to be used by other pages like `bureau-history`?